pyproject.toml

```diff
@@ -20,7 +20,9 @@ dependencies = [
     "packaging",
     "safetensors",
     "dacite",
-    "tqdm"
+    "tqdm",
+    "transformers",
```

Comment:
ah i forgot, we should not require transformers or datasets, because it could be optional

Reply:
Done! Added `nanosets` package to `pyproject.toml` and we install it in `.github/workflows/3d_parallelism_unit_tests.yaml`. Also added to `docs/nanoset.md`. Should we include something in `README.md`?